packages/xxhash/xxhash.0.1/opam

```diff
@@ -15,14 +15,12 @@ install: ["ocaml" "setup.ml" "-install"]
 remove: ["ocamlfind" "remove" "xxhash"]
 depends: [
   "ocaml"
-  "oasis" {build}
-  "ctypes"
+  "oasis" {>= "0.4.7" & build}
+  "ocamlbuild" {!= "0.9.0" & build}
+  "ctypes" {>= "0.6.0"}
```

Comment:
Do you have the logs for these new constraints for our record?

Reply:
I used the CI to determine this. But I force pushed over old commits so I cannot find them any more. But in summary:
- OCamlbuild 0,9.0 does not work for unknown reasons. Version 0 and version 0.9.1 do work https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/6d7deffe78d9ee9d1c220cac83c9a82969174fc1/variant/compilers,4.03,xxhash.0.1,lower-bounds
- Ctypes 0.6.0 introduces `Unsigned.UInt.to_int64` needed by this package: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/76f25ec3012a7834e3d10c40819fba6cda8cc389/variant/compilers,4.04,xxhash.0.1,lower-bounds
- I've already forgotten what the issue with older versions of Oasis was, but I think it may have been this log: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/fcf9f161fc8c67d08665fbcff0dd3c3ff4ce4297/variant/compilers,4.02,xxhash.0.1,lower-bounds

Reply:
```suggestion
  "oasis" {build}
  "ocamlbuild" {!= "0.9.0" & build}
  "ctypes"
```

Reply:
Pretty sure that the ctypes constraint is correct. The `integers` package was split off from `ctypes`, but for older versions it was still part of `ctypes`, and that is the case for 0.6.0